docs/templates/helpers.js

```diff
@@ -9,7 +9,9 @@ module.exports['readme-path'] = opts => {
 module.exports.names = params => params.map(p => p.name).join(', ');
 
 module.exports['typed-params'] = params => {
-  return params.map(p => `${p.type}${p.name ? ' ' + p.name : ''}`).join(', ');
+  console.log(params);
+  if (params == undefined) return [];
+  return params.map(p => `${p.type}${p.indexed ? ' indexed' : ''}${p.name ? ' ' + p.name : ''}`).join(', ');
```

Comment:
With `?.` if params is undefined we will return undefined, which should work more or less the same as the empty array.

```suggestion
  return params?.map(p => `${p.type}${p.indexed ? ' indexed' : ''}${p.name ? ' ' + p.name : ''}`).join(', ');
```